Lib/concurrent/futures/thread.py

```diff
@@ -215,10 +215,25 @@ def _initializer_failed(self):
                 if work_item is not None:
                     work_item.future.set_exception(BrokenThreadPool(self._broken))
 
-    def shutdown(self, wait=True):
+    def shutdown(self, wait=True, cancel_futures=False):
```

Comment:
You forgot to make this one keyword-only

Reply:
Oops, good catch.